srcpkgs/netatalk/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'netatalk'
+pkgname=netatalk
+version=3.1.15
+revision=1
+archs='x86_64'
```

Comment:
This should probably not be restricted to only one architecture.

Reply:
I've tried `-a aarch64` and `-a armv6l-musl`, both of them failed at the configure stage

```
checking for SSL... no
checking for Berkeley DB headers in /usr/local/include/... no
checking for Berkeley DB headers in /usr/local/include/db6.1... no
checking for Berkeley DB headers in /usr/local/include/db6... no
checking for Berkeley DB headers in /usr/local/include/db5... no
checking for Berkeley DB headers in /usr/local/include/db5.3... no
checking for Berkeley DB headers in /usr/local/include/db5.2... no
checking for Berkeley DB headers in /usr/local/include/db5.1... no
checking for Berkeley DB headers in /usr/local/include/db51... no
checking for Berkeley DB headers in /usr/local/include/db5.0... no
checking for Berkeley DB headers in /usr/local/include/db50... no
checking for Berkeley DB headers in /usr/local/include/db4.8... no
checking for Berkeley DB headers in /usr/local/include/db48... no
checking for Berkeley DB headers in /usr/local/include/db4.7... no
checking for Berkeley DB headers in /usr/local/include/db47... no
checking for Berkeley DB headers in /usr/local/include/db4.6... no
checking for Berkeley DB headers in /usr/local/include/db46... no
checking for Berkeley DB headers in /usr/local/include/db4... no
checking for Berkeley DB headers in /usr/include/... no
checking for Berkeley DB headers in /usr/include/db6.1... no
checking for Berkeley DB headers in /usr/include/db6... no
checking for Berkeley DB headers in /usr/include/db5... no
checking for Berkeley DB headers in /usr/include/db5.3... no
checking for Berkeley DB headers in /usr/include/db5.2... no
checking for Berkeley DB headers in /usr/include/db5.1... no
checking for Berkeley DB headers in /usr/include/db51... no
checking for Berkeley DB headers in /usr/include/db5.0... no
checking for Berkeley DB headers in /usr/include/db50... no
checking for Berkeley DB headers in /usr/include/db4.8... no
checking for Berkeley DB headers in /usr/include/db48... no
checking for Berkeley DB headers in /usr/include/db4.7... no
checking for Berkeley DB headers in /usr/include/db47... no
checking for Berkeley DB headers in /usr/include/db4.6... no
checking for Berkeley DB headers in /usr/include/db46... no
checking for Berkeley DB headers in /usr/include/db4... no
configure: error: Berkeley DB library required but not found!
```